tool/src/kotlin/mod.rs

```diff
@@ -60,8 +61,14 @@ pub(crate) fn run<'tcx>(
 
     let conf_str = std::fs::read_to_string(conf_path)
         .unwrap_or_else(|err| panic!("Failed to open config file {conf_path:?}: {err}"));
-    let KotlinConfig { domain, lib_name } = toml::from_str::<KotlinConfig>(&conf_str)
+    let KotlinConfig {
+        domain,
+        lib_name,
+        use_finalizers_instead_of_cleaners,
+    } = toml::from_str::<KotlinConfig>(&conf_str)
         .expect("Failed to parse config. Required fields are `domain` and `lib_name`");
+    let use_finalizers_instead_of_cleaners =
```

Comment:
`use_finalizers_or_cleaners.unwrap_or(false)`

Reply:
nice, that's clean